locales/base/translation.json

```diff
@@ -2014,7 +2014,11 @@
     "finiteApprovalDescription": "Approved {{approvedAmount}} {{tokenSymbol}} for trade",
     "infiniteApprovalDescription": "Approved infinite {{tokenSymbol}} for trade",
     "revokeApprovalDescription": "Revoked {{tokenSymbol}} for trade",
-    "descriptionLabel": "Details"
+    "descriptionLabel": "Details",
+    "getStarted": {
```

Comment:
nit: not sure I would add this under transactionFeed, seems like this content is shown as an alternative to the feed, rather than as a subset of it

Reply:
Moved in 041a9dd6e624d6e2d1807ca3959f41328fa6f8b0!